dex-it/src/test/scala/com/wavesplatform/it/sync/MatcherRestartTestSuite.scala

```diff
@@ -1,33 +1,61 @@
 package com.wavesplatform.it.sync
 
+import com.wavesplatform.dex.api.http.entities.HttpOrderStatus
 import com.wavesplatform.dex.api.http.entities.HttpOrderStatus.Status
 import com.wavesplatform.dex.domain.order.{Order, OrderType}
 import com.wavesplatform.it.MatcherSuiteBase
+import org.scalacheck.Gen
 
 class MatcherRestartTestSuite extends MatcherSuiteBase {
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     broadcastAndAwait(IssueEthTx)
+    broadcastAndAwait(IssueWctTx)
```

Comment:
Here we send a transaction and wait a block and do this for the second transaction.

Let's send two transaction simultaneously:
```scala
broadcastAndAwait(IssueEthTx, IssueWctTx)
```
and wait a block with them.